tach.toml

```
@@ -0,0 +1,83 @@
+exclude = []
+source_roots = [
+    "model/atmosphere/advection/src",
+    "model/atmosphere/diffusion/src",
+    "model/atmosphere/dycore/src",
+    "model/atmosphere/subgrid_scale_physics/microphysics/src",
+    "model/common/src",
+    "model/driver/src",
+    "tools/src",
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.advection"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.diffusion"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.dycore"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.subgrid_scale_physics.microphysics"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.common"
+depends_on = []
+
+[[modules]]
+path = "icon4py.model.driver"
+depends_on = [
```

you could already add here 
```suggestion
depends_on = [
{path = "icon4py.model.atmosphere.advection`},
{path = "icon4py.model.atmosphere.subgrid_scale_physics.microphysics"}
```
Those are not there yet, but they are allowed.